pom.xml

```diff
@@ -49,6 +50,54 @@
     </license>
   </licenses>
 
+  <scm>
+    <connection>scm:git:https://github.com/apache/apex-core.git</connection>
+    <developerConnection>scm:git:[fetch=]https://github.com/apache/apex-core.git[push=]https://git-wip-us.apache.org/repos/asf/apex-core.git</developerConnection>
+    <url>https://github.com/apache/apex-core.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <issueManagement>
+    <system>jira</system>
+    <url>https://issues.apache.org/jira/projects/APEXCORE</url>
+  </issueManagement>
+
+  <mailingLists>
+    <mailingList>
+      <name>Apex Dev</name>
+      <subscribe>dev-subscribe@apex.apache.org</subscribe>
+      <unsubscribe>dev-unsubscribe@apex.apache.org</unsubscribe>
+      <post>dev@apex.apache.org</post>
+      <archive>http://mail-archives.apache.org/mod_mbox/apex-dev/</archive>
+      <otherArchives>
+        <otherArchive>https://lists.apache.org/list.html?dev@apex.apache.org</otherArchive>
+        <otherArchive>https://s.apache.org/apex-dev</otherArchive>
+      </otherArchives>
+    </mailingList>
+    <mailingList>
+      <name>Apex User</name>
+      <subscribe>users-subscribe@apex.apache.org</subscribe>
+      <unsubscribe>users-unsubscribe@apex.apache.org</unsubscribe>
+      <post>users@apex.apache.org</post>
+      <archive>http://mail-archives.apache.org/mod_mbox/apex-users/</archive>
```

Comment:
http://apex.apache.org/community.html#mailing-lists still points to mod_mbox as a primary archive. Should we change both?